week-1/fix/median.js

```diff
@@ -3,9 +3,22 @@
 // If you're in the week-1 directory, you can run npm test -- fix to run the tests in the fix directory
 
 function calculateMedian(list) {
-  const middleIndex = Math.floor(list.length / 2);
-  const median = list.splice(middleIndex, 1)[0];
-  return median;
+  const listToSort = list.sort((a, b) => a - b);
+  console.log(listToSort);
+  const middleIndex = Math.floor(listToSort.length / 2);
+  console.log(middleIndex);
+  const elementInMiddle = listToSort.slice(middleIndex - 1, middleIndex + 1);
+  console.log(elementInMiddle);
+
+  //for (let i = 0; i < listToSort.length; i++) {
+  if (listToSort.length % 2 === 0) {
+    return (elementInMiddle[0] + elementInMiddle[1]) / 2;
+  } else {
+    return listToSort[middleIndex];
+    //}
```

Comment:
Good work mate, but can you please remove all the commented codes and the all the console.logs. One of the great prices of clean code is to remove all the logs and commented codes